lib/v2/datasource/remote/model/referred_accounts_model.dart

```diff
@@ -0,0 +1,11 @@
+class ReferredAccounts {
+  final List<String> accounts;
+
+  ReferredAccounts({this.accounts = const []});
```

Comment:
do we need default here? 
Can we make the accounts param required ?

Reply:
it was changed to required 👍